examples/speaker_tasks/diarization/README.md

```diff
@@ -131,12 +131,13 @@ In `./demo_asr_output/`, you can check the results as below.
     └── my_audio1.json
     └── my_audio1.txt
     └── my_audio1.rttm
+    └── my_audio1_gecko.json
 │
 └── ...
 ```
 
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  `*.json` files contains word-by-word json output with speaker label and time stamps.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  `*.json` files contains word-by-word json output with speaker label and time stamps. We do also provide json output file to view output of our diarization results in nicely formated [gecko](https://gong-io.github.io/gecko/) tool.
```

Comment:
I suggest to change this line something like this ("nicely formatted" does not sound informative) :

We also provide  json output file for [gecko](https://gong-io.github.io/gecko/) tool where you can visualize the diarization result along with ASR output.

Reply:
updated